bedrock/firefox/firefox_details.py

```diff
@@ -120,7 +120,7 @@ def _get_filtered_builds(self, builds, channel, version=None, query=None):
                 build_info['platforms'][platform] = {
                     'download_url': self.get_download_url(channel, version,
                                                           platform, locale,
-                                                          True),
+                                                          True, True),
```

Comment:
Yes, this is in `_get_filtered_builds` and the function is only for the /all/ pages. The download buttons are not affected.